### ISSUE TITLE
[Issue]: Can't update, asks for an email address

### ISSUE THREAD

**Issue by wzol** [Nevysha/Cozy-Nest#173]
### What happened?

Using normally and suddenly it gives me this:

Committer identity unknown

```
*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal: unable to auto-detect email address
```

No idea what this is, I'm too novice to do extra tweaking. Also in the top right corner it says it wants to update, but clicking the update button does nothing.

### Webui

Automatic1111's webui

### What browsers are you seeing the problem on?

Microsoft Edge

### Additional info

A bug happened!

### Checklist

- [X] I have searched the issues to make sure this is not a duplicate
- [X] Webui is up to date
- [X] Cozy-Nest is up to date
- [X] All my extensions are up to date

**Comment by Nevysha** [comment]
>No idea what this is
This is from git. Have you modified some source file of Cozy Nest ?

**Comment by wzol** [comment]
No, nothing like that. But it seems, that deleting and reinstalling fixed it. Noticed only one weird thing yet, textual inversion always loads into the prompt, even if the cursor is the negative prompt. Not sure if this is related.

**Comment by wzol** [comment]
Another suspicious thing, the right size is messed up with sd-webui-prompt-history - is it an incompatibility, or something is really messed up on my side?

**Comment by Nevysha** [comment]
>No, nothing like that. But it seems, that deleting and reinstalling fixed it. Noticed only one weird thing yet, textual inversion always loads into the prompt, even if the cursor is the negative prompt. Not sure if this is related.

Yes I have not implemented this feature yet

>sd-webui-prompt-history

Not compatible atm but I'll work on it eventually
